src/test/appdistro/client.spec.ts

```diff
@@ -39,17 +41,44 @@ describe("distribution", () => {
 
   describe("getApp", () => {
     it("should throw error when app does not exist", () => {
-      nock(api.appDistributionOrigin).get(`/v1alpha/apps/${appId}`).reply(404, {});
+      nock(api.appDistributionOrigin)
+        .get(`/v1alpha/apps/${appId}?appView=${appViewBasic}`)
+        .reply(404, {});
       return expect(appDistributionClient.getApp()).to.be.rejected;
     });
 
     it("should resolve when request succeeds", () => {
-      nock(api.appDistributionOrigin).get(`/v1alpha/apps/${appId}`).reply(200, {});
+      nock(api.appDistributionOrigin)
+        .get(`/v1alpha/apps/${appId}?appView=${appViewBasic}`)
```

Comment:
Optionally, if it's easier to understand/manage, you can use `.query({})` in nock to specify the query to match: https://www.npmjs.com/package/nock#specifying-request-query-string

Reply:
Good idea, I'll update